pkg/core/tekton/builder/task_spec.go

```diff
@@ -0,0 +1,71 @@
+package builder
+
+import (
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TaskSpecBuilder holds the tekton task spec definition.
+type TaskSpecBuilder struct {
+	TaskSpec v1beta1.TaskSpec
+}
+
+// NewTaskSpecBuilder creates a TaskSpec with default values.
+func NewTaskSpecBuilder(name, image, command string) *TaskSpecBuilder {
+	b := &TaskSpecBuilder{}
+	b.TaskSpec = v1beta1.TaskSpec{
+		Steps: []v1beta1.Step{{Container: corev1.Container{
+			Name:    name,
+			Image:   image,
+			Command: []string{command},
+		}}},
+	}
+	return b
+}
+
+// TaskParam adds a ParamSpec to the TaskSpec.
+func (b *TaskSpecBuilder) TaskParam(name string, description ...string) {
```

Comment:
Same here: `Task` prefix may be stripped.

Reply:
done.